config/default.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Nice work, you've just exposed your database.
Consider deleting the old database and creating a new one.
Then exclude this file from .git using .gitignore.
Otherwise, expect to see a huge bill from MLab.